app/src/main/res/layout-land/add_profile_activity.xml

```diff
@@ -218,7 +219,7 @@
                 android:id="@+id/add_profile_activity_allow_download_switch"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:clickable="@{viewModel.validPin ? true : false}"
+                android:enabled="@{viewModel.validPin ? true : false}"
```

Comment:
Out of curiosity, does just ``android:enabled=@{viewModel.validPin}`` work?

Reply:
Yes it does work correctly.